test/lib/ansible_test/_internal/sanity/__init__.py

```diff
@@ -286,7 +286,8 @@ def __init__(self, args):  # type: (SanityConfig) -> None
                 self.parse_errors.append((line_no, 1, "Line cannot be empty or contain only a comment"))
                 continue
 
-            parts = line.split(' ')
+            parts = line.replace('  ', ' ')
```

Comment:
This fix does not seem robust enough. What happens in cases where there are ten spaces?

This should probably be a regxp to replace two or more spaces, or continue to be strict about the formatting.

The consensus from discussion today was that continuing to be strict about the formatting.